spec/support/user_login.rb

```diff
@@ -0,0 +1,14 @@
+def login_as(user)
+  visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.